package.json

```diff
@@ -29,7 +29,6 @@
   "homepage": "https://github.com/bringg/node-arnavmq#readme",
   "dependencies": {
     "amqplib": "^0.10.3",
-    "p-defer": "^3.0.0",
```

Comment:
we actually had a [own version](https://github.com/bringg/node-arnavmq/pull/21/files#diff-b42e79919256b6c071b190d2d787af36d7da1734294f325b17c1f33bbc4fdbc0) of this, but deciddedd to use this package to avoid dmanaging custom version, so please lets keep it.

Reply:
No management is required in this it's just 4 lines of simple code...

Reply:
as explained earlies we had this already and it was decided to remove from the codebase, `p-defer`  is commonly used package and download 8 million times a day. Lets please stik with it.